src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -610,13 +612,14 @@ public File createApprovedDARDocument() throws IOException {
             for (Election election : elections) {
                 Document dar = getDataAccessRequestByReferenceIdAsDocument(election.getReferenceId());
                 DataAccessRequest dataAccessRequest = findByReferenceId(election.getReferenceId());
+                User user = userDAO.findUserById(dataAccessRequest.getUserId());
                 try {
-                    if (dar != null) {
+                    if (dar != null && user != null) {
                         Integer datasetId = dataAccessRequest.getData().getDatasetIds().get(0);
                         String consentId = dataSetDAO.getAssociatedConsentIdByDataSetId(datasetId);
                         Consent consent = consentDAO.findConsentById(consentId);
-                        String profileName = userPropertyDAO.findPropertyValueByPK(dataAccessRequest.getUserId(), DarConstants.PROFILE_NAME);
-                        String institution = userPropertyDAO.findPropertyValueByPK(dataAccessRequest.getUserId(), DarConstants.INSTITUTION);
+                        String profileName = user.getDisplayName();
+                        String institution = institutionDAO.findInstitutionById(user.getInstitutionId()).getName();
```

Comment:
This could use a null check on the institution. If we don't have one, log a warning and return the empty string.